# HG changeset patch
# User Doug Stewart <doug.dastew@gmail.com
# Date 1417316091 18000
#      Sat Nov 29 21:54:51 2014 -0500
# Node ID cc7931e8953d9739b69aec069e47850515292085
# Parent  2218dd129a6be320ef9a77640db7462489c8204b
lscov.m: increase test tolerance so that they pass (bug #43118)

* lscov.m: set tolerance to eps for all tests.

diff --git a/scripts/statistics/base/lscov.m b/scripts/statistics/base/lscov.m
--- a/scripts/statistics/base/lscov.m
+++ b/scripts/statistics/base/lscov.m
@@ -167,20 +167,20 @@ endfunction
 %! V = .2*ones(length(x1)) + .8*diag(ones(size(x1)));
 %! [bg, sew_b, mseg] = lscov (X, y, V);
 %! assert(bg, [0.1203 0.3284 -0.1312]', 1E-4);
 %! assert(sew_b, [0.0672 0.2267 0.1488]', 1E-4);
 %! assert(mseg, 0.0019, 1E-4);
 %! y2 = [y 2*y];
 %! [b2, se_b2, mse2, S2] = lscov (X, y2);
 %! assert(b2, [b 2*b], eps)
-%! assert(se_b2, [se_b 2*se_b])
-%! assert(mse2, [mse 4*mse])
-%! assert(S2(:, :, 1), S)
-%! assert(S2(:, :, 2), 4*S)
+%! assert(se_b2, [se_b 2*se_b], eps)
+%! assert(mse2, [mse 4*mse], eps)
+%! assert(S2(:, :, 1), S, eps)
+%! assert(S2(:, :, 2), 4*S, eps)
 
 %!test
 %! ## Artificial example with positive semidefinite weight matrix
 %! x = (0:0.2:2)';
 %! y = round(100*sin(x) + 200*cos(x));
 %! X = [ones(size(x)) sin(x) cos(x)];
 %! V = eye(numel(x));
 %! V(end, end-1) = V(end-1, end) = 1;
